unofficial/c511027107.lua

```diff
@@ -0,0 +1,144 @@
+--ＺＷ－極星神馬聖鎧
+--ZW - Sleipnir Mail (anime)
+--Script by Rundas
+local s,id=GetID()
+function s.initial_effect(c)
+	--equip
+	local e1=Effect.CreateEffect(c)
+	e1:SetDescription(aux.Stringid(id,0))
+	e1:SetType(EFFECT_TYPE_IGNITION)
+	e1:SetProperty(EFFECT_FLAG_CARD_TARGET)
+	e1:SetCategory(CATEGORY_EQUIP)
+	e1:SetRange(LOCATION_MZONE)
+	e1:SetTarget(s.eqtg)
+	e1:SetOperation(s.eqop)
+	c:RegisterEffect(e1)
+	--change atk on attack
+	local e2=Effect.CreateEffect(c)
+	e2:SetDescription(aux.Stringid(id,1))
+	e2:SetCategory(CATEGORY_ATKCHANGE)
+	e2:SetType(EFFECT_TYPE_QUICK_O)
+	e2:SetCode(EVENT_BATTLE_CONFIRM)
+	e2:SetRange(LOCATION_SZONE)
+	e2:SetCondition(s.atkcon)
+	e2:SetCost(s.cost)
+	e2:SetOperation(s.atkop)
+	c:RegisterEffect(e2)
+    --special summon
+	local e3=Effect.CreateEffect(c)
+	e3:SetDescription(aux.Stringid(id,2))
+	e3:SetType(EFFECT_TYPE_SINGLE+EFFECT_TYPE_TRIGGER_O)
+	e3:SetCategory(CATEGORY_SPECIAL_SUMMON)
+	e3:SetProperty(EFFECT_FLAG_DAMAGE_STEP)
+	e3:SetCode(EVENT_LEAVE_FIELD)
+	e3:SetCondition(s.spcon)
+	e3:SetTarget(s.sptg)
+	e3:SetOperation(s.spop)
+	c:RegisterEffect(e3)
+end
+
+s.listed_names={56840427}
+
+--equip
+
+function s.filter(c)
+	return c:IsFaceup() and c:IsCode(56840427)
+end
+
+function s.eqtg(e,tp,eg,ep,ev,re,r,rp,chk,chkc)
+	if chkc then return chkc:IsLocation(LOCATION_MZONE) and chkc:IsControler(tp) and s.filter(chkc) end
+	if chk==0 then return Duel.GetLocationCount(tp,LOCATION_SZONE)>0
+		and Duel.IsExistingTarget(s.filter,tp,LOCATION_MZONE,0,1,nil) end

```

Comment:
You should carry over the changes from pyrQ's suggestion
Also, you are missing SetOperationInfo in ```eqtg```